hack/scripts/remove_ovs_bridges.py

```diff
@@ -0,0 +1,85 @@
+import subprocess
+import os
+import re
+import requests
+
+# step 1: get all ovs bridges:
```

Comment:
can you also Readme.md on how to run these scripts and brief description on what each script does